code/modules/modular_computers/hardware/hard_drive.dm

```diff
@@ -164,6 +164,13 @@
 	stored_files = null
 	return ..()
 
-/obj/item/weapon/computer_hardware/hard_drive/New()
+/obj/item/weapon/computer_hardware/hard_drive/Initialize(mapload)
```

Comment:
I think is not being called ever, yes?

Reply:
It works? So I don't see what the issue is.